torch/onnx/utils.py

```diff
@@ -1251,12 +1251,10 @@ def unconvertible_ops(
     return graph, unsupported_ops
 
 
-def _setup_trace_module_map(model, export_modules_as_functions):
-    def __setup_trace_module_map():
-        trace_module_map = {_m: torch.typename(type(_m)) for _m in model.modules()}
-        torch.jit._trace._trace_module_map = trace_module_map
-        return trace_module_map
-
+def _setup_trace_module_map(
+    model: Union[torch.nn.Module, torch.jit.ScriptModule],
+    export_modules_as_functions: Union[bool, Collection[Type[torch.nn.Module]]],
+):
```

Comment:
```suggestion
) -> bool:
```
does it return bool?

Reply:
I think it's not. From the if and elif branch, it seems like could be dict.

Reply:
updated to `Set[str]`